test/e2e/aks_autoscaler.go

```diff
@@ -0,0 +1,125 @@
+//go:build e2e
+// +build e2e
+
+/*
+Copyright 2022 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
```

Comment:
instead of having a bunch of aks_ files, what are your thoughts on adding an `aks` sub folder?

Reply:
That's a slightly more advanced operation but if that has more long-term maintenance benefits, happy to do that.

@mboersma @nojnhuh thoughts here?

Reply:
I don't see any strong reasons to prefer one option over the other. I can see where a subdirectory would keep the file tree a bit tidier, but readability in azure_test.go might take a marginal hit if the AKS test specs need a package prefix. In that case if we had something like `AzureLBSpec` and `aks.AutoscaleSpec` used similarly, I might assume at first glance that they're more different conceptually than they actually are based on one being in a sub-package. That's quite a bit of "reading between the lines" though.

Overall I think I'm leaning slightly toward keeping the AKS files in this existing directory and not putting them in their own.

Reply:
To clarify I don't have any strong arguments for adding a folder, just pointing it out since it jumped at me when seeing all the different files with the same prefix. Either option is fine with me.